src/views/Setup.vue

```diff
@@ -3,9 +3,16 @@ import { ref } from 'vue';
 import { useRouter } from 'vue-router';
 
 const router = useRouter();
+const tlds = ['.eth', '.xyz', '.com', '.org', '.io', '.app', '.art'];
 
 const id = ref('');
 
+function isDisabledTLD(id) {
```

Comment:
I would rather have a isValid function instead of isInvalid
```suggestion
function isValidTLD(id) {
```